ansible_collections/arista/avd/docs/release-notes/3.x.x.md

```diff
@@ -2,6 +2,14 @@
 
 Documentation for AVD version `3.x.x` [available here](https://www.avd.sh/en/releases-v3.x.x/)
 
+## Release 3.3.3
+
+### Fixed issues
+
+- d0e081944 Fix: Set requirement for ansible.netcommon to <2.6 (#1557)
+- 177957356 Fix(eos_designs): Feature interaction issues with evpn_services_l2_only (#1556)
+- f60a7f5db Doc(eos_designs): Fix missing max_parallel_uplinks (#1551)
+
```

Comment:
I think we fixed the doc related to qos priority strict.

Reply:
Not really sure because I didn't go through the complete update.

Reply:
I only cherry-picked these 3 commits + Release-notes. The Qos one was a feature PR, where the documentation was also improved, so that will be included only in the next feature release.